From 060e0883f0a67ad4197704fe6433c78048ab6768 Mon Sep 17 00:00:00 2001
From: Iceows <mounierr07@gmail.com>
Date: Sat, 7 Oct 2023 22:51:10 +0200
Subject: [PATCH 2/3] rw-system: huawei - audio custom use rpc_motion_req

Fix this error :
audio_custom: notify_sensor_call_state, open /sys/class/sensors/rpc_sensor/rpc_motion_req failed, fd = -1
---
 rw-system.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/rw-system.sh b/rw-system.sh
index 67ac152..910725b 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -615,6 +615,9 @@ if getprop ro.vendor.build.fingerprint | grep -iq -E -e 'huawei|honor' || getpro
         mount -o bind /product/etc/libnfc-nxp.conf /system/etc/libnfc-nxp.conf || true
 
     mount -o bind /system/phh/libnfc-nci-huawei.conf /system/etc/libnfc-nci.conf
+
+	# For audio_custom
+	chown system:system /sys/class/sensors/rpc_sensor/rpc_motion_req
 fi
 
 if getprop ro.vendor.build.fingerprint | grep -qE -e ".*(crown|star)[q2]*lte.*" -e ".*(SC-0[23]K|SCV3[89]).*" && [ "$vndk" -lt 28 ]; then
-- 
2.33.0.windows.2

